common/pulp_docker/common/tarutils.py

```diff
@@ -114,3 +114,24 @@ def get_youngest_children(metadata):
                 pass
 
     return list(image_ids)
+
+
+def get_image_manifest(tarfile_path):
+    """
+    Given a path to a tarfile, which itself is the product of "skopeo copy"
+    this returns the decoded manifest.json file if it exists.
+
+    :param tarfile_path: full path to tarfile
+    :type tarfile_path:  basestring
+    :return:             decoded manifest json
+    :rtype               list or dict
+    """
+    image_manifest = []
```

Comment:
This is the output of `docker save` on the busybox image.
```
├── 6803c618d786ea5b542ba692120df35e59c4592ad7a4138450b2e2277088d15a
│   ├── json
│   ├── layer.tar
│   └── VERSION
├── efe10ee6727fe52d2db2eb5045518fe98d8e31fdad1cbdd5e1f737018c349ebb.json
├── manifest.json
└── repositories
```
It does contain the manifest file. But, the busyboxlight.tar file which is kept in the `test/data` folders doesn't contain a manifest file. That's why I went for this approach. Let me know what you think of it.